src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -1,12 +1,23 @@
 package core.basesyntax;
 
+import java.time.DateTimeException;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Locale;
 import java.util.Optional;
 
 public class JavaDateTimeApi {
+    private static LocalDate CURRENT_DATE = LocalDate.now();
+    private static String UKRAINE_ZONE = "+02:00";
```

Comment:
what is wrong?

Reply:
Need to make ZoneOffset UKRAINE_ZONE = ZoneOffset.of("+02:00")

Reply:
@KostiantynVorobiov what modifiers should a constant have?

Reply:
final, I forgot about this